src/FNA3D_CommandBuffer.c

```diff
@@ -225,24 +221,17 @@ static void FNA3D_CommandBuffer_CleanContainer(
 		transferBuffer = container->transferBuffers[i];
 		transferBuffer->offset = 0;
 
-		if (transferBuffer == manager->transferBufferPool.fastTransferBuffer)
+		if (manager->transferBufferPool.availableTransferBufferCount == manager->transferBufferPool.availableTransferBufferCapacity)
 		{
-			manager->transferBufferPool.fastTransferBufferAvailable = 1;
+			manager->transferBufferPool.availableTransferBufferCapacity += 1;
```

Comment:
i'm guessing we have so few of these that growing by 1 is fine instead of by larger amounts or a ratio?

Reply:
yeah, these are allocated infrequently